netmanager/package-lock.json

```diff
@@ -10450,7 +10450,7 @@
             "integrity": "sha512-J9X76xnncMw+wIqb15HeWfPMqPwYxSpPY8yWPJ7rAZN/ZDzFkjCSZObryCyUe8zbrVRNiuCnIeQteCzMn7GnWw==",
```

Comment:
This file change should not be part of this PR.

Reply:
Working on that